source/options.css

```diff
@@ -69,15 +69,26 @@ h2 ~ h2 {
 	text-decoration: line-through;
 }
 
+.js-features :not(:checked) + .info .extended-options {
+	display: none;
+}
+
 .js-features .description {
-	font-style: 0.8em;
```

Comment:
👎

Reply:
My editor was bugging me 😅.

And it's supposed to be `font-size`, not `font-style`.

Reply:
Oh wow, it wasn’t being caught by stylelint either. 😳 

Can you restore it as font-size? I tested the value in the developer tools and then wrote it this way in the repo 🤦‍♂️

Reply:
On Chrome the `font-size` for this is already at `12px`, if we add this rule, then it goes down to `9.6px`, way too small for anyone to read.

The current options UI font size if fine, IMHO.